README.md

```diff
@@ -125,6 +125,31 @@ rspec packs/foobar/spec
 rspec packs/foobar/nested_pack
 ```
 
+#### factory_bot and factory_bot_rails
+
+Ensure you have the gem in your Gemfile, and that `require: false` is not present, otherwise the `packs-rails` FactoryBot integration will not be run upon application/environment load.
```

Comment:
At least in my case it was specifically the `gem 'factory_bot_rails', require: false`, I left the `factory_bot` with the require and it works fine, could you give it a try so we can narrow the problem even more?

Reply:
For me I actually don't have `factory_bot` specifically listed in the gemfile because it's already a dependency of `factory_bot_rails` - but I can add that detail thank you

Reply:
Now that you mention it I'm even unclear on how different `require` true/false would work when `factory_bot_rails` interacts with `factory_bot`